lib/ansible/module_utils/urls.py

```diff
@@ -550,7 +550,10 @@ def build_ssl_validation_error(hostname, port, paths):
     msg.append('You can use validate_certs=False if you do'
                ' not need to confirm the servers identity but this is'
                ' unsafe and not recommended.'
-               ' Paths checked for this platform: %s')
+               ' Paths checked for this platform: %s.')
+
+    if exc:
+        msg.append('The exception msg was: %s.' % exc)
```

Comment:
I would prefer to see the `to_native()` call done here, rather than as it's passed in below. Fix that and we'll merge this.